packages/adp-tooling/src/writer/index.ts

```diff
@@ -42,26 +49,18 @@ export async function generate(basePath: string, config: AdpWriterConfig, fs?: E
     if (!fs) {
         fs = create(createStorage());
     }
+
     const tmplPath = join(__dirname, '../../templates/project');
     const fullConfig = setDefaults(config);
+    const ignoredFiles = fullConfig.appdescr ? ['**/manifest.appdescr_variant'] : [];
 
-    fs.copyTpl(join(tmplPath, '**/*.*'), join(basePath), fullConfig, undefined, {
-        globOptions: { dot: true },
-        processDestinationPath: (filePath: string) => filePath.replace(/gitignore.tmpl/g, '.gitignore')
-    });
-
-    // ui5.yaml
-    const ui5ConfigPath = join(basePath, 'ui5.yaml');
-    const baseUi5ConfigContent = fs.read(ui5ConfigPath);
-    const ui5Config = await UI5Config.newInstance(baseUi5ConfigContent);
-    enhanceUI5Yaml(ui5Config, fullConfig);
-    fs.write(ui5ConfigPath, ui5Config.toString());
-    // ui5-deploy.yaml
-    if (hasDeployConfig(fullConfig)) {
-        const ui5DeployConfig = await UI5Config.newInstance(baseUi5ConfigContent);
-        enhanceUI5DeployYaml(ui5DeployConfig, fullConfig);
-        fs.write(join(basePath, 'ui5-deploy.yaml'), ui5DeployConfig.toString());
+    writeTemplateToFolder(join(tmplPath, '**/*.*'), join(basePath), fullConfig, fs, ignoredFiles);
+    if (fullConfig.appdescr) {
+        writeManifestAppdescr(tmplPath, basePath, fullConfig.appdescr, fs);
     }
+    await writeUI5DeployYaml(basePath, fullConfig, fs);
+    await writeUI5Yaml(basePath, fullConfig, fs);
+    await writeEnvFile(basePath, fullConfig, fs);
```

Comment:
Does the user know that we are writing user/password to the filesystem? If not, we cannot do that. 
Btw, as far as I understand, both user/password are only used for the additional tasks for S/4HANA Cloud Public Edition scenarios, however, such systems won't allow access with user/password, therefore, I'd strongly prefer to remove this user/password/env thing.

Reply:
As discussed, please remove it.

Reply:
Done